src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1104,23 +1103,11 @@ public ImmutableArray<SymbolDisplayPart> ToDisplayParts(SymbolDisplayFormat form
             CSharpSyntaxNode block,
             CSharpSyntaxNode expression,
             CSharpSyntaxNode syntax,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnostics,
+            ErrorCode code)
```

Comment:
We use default values in many methods. Besides, the consequence of having a default value which is used incorrectly is that the error will be `CS8057: Methods and accessors cannot combine block bodies with expression bodies` which doesn't sound that bad.